webapp/packages/core-blocks/src/FormControls/Filter.tsx

```diff
@@ -16,12 +16,12 @@ import filterStyle from './Filter.m.css';
 import { InputField } from './InputField';
 
 interface BaseProps {
-  toggleMode?: boolean;
   placeholder?: string;
   disabled?: boolean;
   max?: boolean;
   className?: string;
-  onToggle?: (status: boolean) => void;
+  onFilter?: (value: string) => void;
+  onClean?: () => void;
```

Comment:
what difference between `onFilter` and `onChange`?
is `onClean` is the same as `onChange` with `value === ''`

Reply:
onFilter triggers only on filter button click in manual mode. onChange fires every time
What about onClean, we want to know when the user clean the input by pressing clean button,  not when he deletes everything from input by pressing delete for example